### PR TITLE
Added build for apple arm (silicon) processors

### PR DESCRIPTION
**Pull request by grolu**
**What this PR does / why we need it**:
Added build for apple arm (silicon) processors

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```feature user

```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -40,16 +40,20 @@ lint: ## Run golangci-lint against code.
 ##@ Build
 
 .PHONY: build
-build: build-darwin build-linux build-windows ## Build gardenctl binary for darwin, linux and windows.
+build: build-darwin-amd64 build-darwin-arm64 build-linux build-windows ## Build gardenctl binary for darwin, linux and windows.
```
Comment (petersutter) [inline Makefile:43]: let's also adapt the other make targets and add the arch there

**Review by petersutter — APPROVED** [review]
/lgtm

File: Makefile (hunk 1 above)
Comment (grolu) [reply Makefile:43]: I forgot those

**Review by petersutter — APPROVED** [review]
/lgtm

**Pull request merged**